crowbar_framework/app/barclamps/hive_barclamp.rb

```diff
@@ -15,17 +15,28 @@
 # limitations under the License.
 #
 
-barclamp:
-  name: sqoop
-  display: sqoop
-  version: 0
-  requires:
-    - hadoop
-  member:
-    - apachehadoop
-
-crowbar:
-  layout: 1
-  order: 500
-  run_order: 500
-  chef_order: 500
+class HiveBarclamp < Crowbar::Registry::Barclamp
+  name "hive"
+  display "Hive"
+  description "Hadoop Hive component"
+
+  member [
+    "apachehadoop"
+  ]
+
+  requires [
+    "hadoop"
+  ]
+
+  listed true
+
+  layout 1
+  version 0
+  schema 3
+
+  order 400
+
+  nav(
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.